src/store/index.js

```diff
@@ -18,6 +20,15 @@ export default new Vuex.Store({
     },
     updateGeolocation (state, geolocation) {
       state.geolocation = geolocation
+    },
+    storeEmail (state, email) {
+      state.email = email
+    },
+    clearResults (state) {
+      state.searchResults = []
+    },
+    changeToSearch (state) {
+      state.homeChildComp = 'FindPark'
```

Comment:
Glad we were able to control it this way. We can revisit this later and refine it.

Reply:
Yeah, this doesn't seem like data that _needs_ to exist in the store.  We'll have to consider a different way to manage the data locally that allows the user to search without re-entering the email address each time that view is visited.